homeassistant/components/modbus/const.py

```diff
@@ -84,3 +93,11 @@
 }
 DEFAULT_TEMP_UNIT = "C"
 MODBUS_DOMAIN = "modbus"
+
+CONF_PLATFORMS = (
```

Comment:
We only use `CONF_` prefix for config key constants.
```suggestion
PLATFORMS = (
```

Reply:
Done.